app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for exceptions related to vaccination."""
```

Comment:
These docstrings provide pretty much same information that is in excpetion name so why do we need them here?

Reply:
В теорії було вказано, що докстрінг це гарна практика
+ в усьому, що імпортується(бібліотеки) є докстрінги з уточненнями. 
Якщо в цьому проєкті це заборонено, можу прибрати

Reply:
Hey, good point - it is not prohibited to use docstrings. My point was that not all exception classes need docstrings if their name is self-explanatory. For example 
```
NotWearingMaskError = Exception raised when a visitor is not wearing a mask

So it's not really a mistake, in real life it may be project-specific